anyblok_furetui/furetui/crud.py

```diff
@@ -94,6 +94,7 @@ def read(cls, request):
             request.authenticated_userid
         )
         query = qs.from_filter_by(query)
+        query = qs.from_filter_by_primary_keys(query)
```

Comment:
add query = qs.from_composite_filter_by(query) where issue 64 of rest api will be merged

Reply:
+1

Reply:
pushed